gnomad/gnomadv4_extract_rows.py

```diff
@@ -0,0 +1,30 @@
+# LT 31/08/2022
+
+## annotate gnomADv4 with frequencies and variant QC standard fields
+## export the rows as a hail.Table on a cpg bucket
+
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+# connect to hail, requester_pays parameters are setup in the dataproc parameters
+hl.init(default_reference='GRCh38')
+
+# gnomAD v4 exomes VDS
+big_vds = hl.vds.read_vds('gs://gnomad/v4.0/raw/exomes/gnomad_v4.0.vds')
+
+# densify as a MatrixTable
+mt = hl.vds.to_dense_mt(big_vds)
+
+# remove REF/REF rows left after densification
+mt = mt.filter_rows(hl.len(mt.alleles) < 2, keep=False)
+
+# get alleles frequencies
+mt = mt.transmute_entries(GT=hl.experimental.lgt_to_gt(mt.LGT, mt.LA))
+mt = hl.variant_qc(mt)
+
+# keep only the rows and repartition
+# given the drastic size reduction by summarizing the columns, reduce the number of partitions ~50 fold
+ht = mt.rows().repartition(1000, shuffle=True)
```

Comment:
A shuffle would require preemptible workers, but you don't really need fully balanced partitions here.

```suggestion
ht = mt.rows().naive_coalesce(1000)
```

However, given how problematic repartitioning operations often are, I'd suggest writing out all 50k partitions first and then trying to repartition in a follow-up process, if necessary.